launch/launch/main.py

```diff
@@ -45,9 +45,9 @@ def main(argv=sys.argv[1:]):
     args = parser.parse_args(argv)
 
     launcher = DefaultLauncher()
-    for launch_file in args.launch_file:
+    for launch_file in args.launch_files:
         launch_descriptor = LaunchDescriptor()
-        load_launch_file(args.launch_file, launch_descriptor, args.args)
+        load_launch_file(launch_file, launch_descriptor, args.args)
```

Comment:
Only the change in this line is necessary. The rest should stay unchanged.

Reply:
ok, reverted the second commit